### DISCUSSION TITLE
Enhancing a picture without changing it

### DISCUSSION THREAD

**Discussion by RChadwick7** [lllyasviel/Fooocus#3428] [Q&A] [answered]
I have a picture taken 40 years ago from a Disk camera. If Disk cameras were before your time, they took VERY grainy photos. Anyway, I asked Fooocus to clean it up. If I ask Fooocus to create a random person, the results are stunning. I was hoping the same for my poor picture. Unfortunately it didn't remove most noise, the sky was still washed out instead of blue, and details were iffy. I would have called it an improvement, except it changed the face, making it longer, looking like someone totally different. Any advice on how to enhance photos? Is it possible to have Fooocus 're-imagiine' a picture, making it very high quality, yet adhering to the original image as much as possible? Is it also possible to add additional, better quality pictures of the subject so the face doesn't get changed?
Thanks!

**Comment by mashb1t** [comment, marked as answer]
This sounds contradictory, you want to change the image to remove noise but not change it to keep the subject.
What you can do is to use inpaint only for a specific area around the subject, or vary (subtle) on the whole image to do minimal adjustments.
You may provide a high res image of the subject in image prompt and enable mixing in the developer debug mode > control, either with upscale or with inpaint, to add more context.

**Comment by Yogesh-DevHub** [comment]
Hi, there is something similar I made a video about "face change after upscaling", you can fast-forward to 1:08. This might help you.

https://youtu.be/1ih-XVc6JS8?si=MsAXiJ8YEfzkfiUZ

**Comment by kalle07** [comment]
use photoshop RAW -> denoise and denoise color
or try topaz photo-AI

**Comment by RChadwick7** [comment]
Thanks for the replies. The original picture is pretty bad. I don't think there's any saving it, no matter what photoshop magic is run on it. Can Fooocus redraw from scratch an image based not on a prompt, but another picture? Can it redraw the picture?

**Reply by mashb1t** [reply]
> What you can do is to use inpaint only for a specific area around the subject, or vary (subtle) on the whole image to do minimal adjustments.

**Comment by kalle07** [comment]
can you post/upload the image , i can try  ;)
